glide.yaml

```diff
@@ -2,34 +2,34 @@ package: github.com/openshift/apiserver-library-go
 import:
 # kube first
 - package: k8s.io/api
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/client-go
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/component-base
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/kubernetes
-  version: v1.14.0
+  version: v1.16.0-beta.1
+- package: k8s.io/klog
+  version: v0.4.0
```

Comment:
I had to add this dependency explicitly, otherwise, it failed:
```
go build ./...
# github.com/openshift/apiserver-library-go/vendor/k8s.io/utils/trace
vendor/k8s.io/utils/trace/trace.go:100:57: invalid operation: stepThreshold == 0 || stepDuration > stepThreshold || klog.V(4) (mismatched types bool and klog.Verbose)
vendor/k8s.io/utils/trace/trace.go:112:56: invalid operation: stepThreshold == 0 || stepDuration > stepThreshold || klog.V(4) (mismatched types bool and klog.Verbose)
# github.com/openshift/apiserver-library-go/vendor/k8s.io/apiserver/pkg/server/httplog
vendor/k8s.io/apiserver/pkg/server/httplog/httplog.go:159:2: non-bool klog.V(3) (type klog.Verbose) used as if condition
vendor/k8s.io/apiserver/pkg/server/httplog/httplog.go:189:9: non-bool klog.V(2) (type klog.Verbose) used as if condition
# github.com/openshift/apiserver-library-go/vendor/k8s.io/client-go/transport
vendor/k8s.io/client-go/transport/round_trippers.go:70:11: cannot convert klog.V(9) (type klog.Verbose) to type bool
vendor/k8s.io/client-go/transport/round_trippers.go:72:11: cannot convert klog.V(8) (type klog.Verbose) to type bool
vendor/k8s.io/client-go/transport/round_trippers.go:74:11: cannot convert klog.V(7) (type klog.Verbose) to type bool
vendor/k8s.io/client-go/transport/round_trippers.go:76:11: cannot convert klog.V(6) (type klog.Verbose) to type bool
```

Reply:
I think upstream uses the same version - https://github.com/kubernetes/apiserver/commit/785eacbd19aedae1e4700720711381e2693c8cfd#diff-37aff102a57d3d7b797f152915a6dc16R59